xkcd/xkcd.py

```diff
@@ -0,0 +1,54 @@
+from redbot.core import commands
+import discord
+import random
+import asyncio,aiohttp,async_timeout,json
+
+
+#Define function to do web requests
+async def fetchGet(urlIn):
```

Comment:
Can you please make your vars and funcs snake_case to align with pep8? kthxbai

Reply:
4fc53da